src/lib.rs

```diff
@@ -42,30 +41,35 @@ extern crate crypto;
 extern crate rustc_back;
 
 use std::cmp;
+use std::str;
 use rustc_back::tempdir::TempDir;
 use crypto::sha2::Sha512 as Sha512;
 use crypto::digest::Digest;
+
 // this is pub to test the tests dir integration tests these are temp and need to be
 // replaced with actual integration tests and this should be private
 mod encryption;
 /// Information required to recover file content from chunks
 pub mod datamap;
+
 /// MAX_CHUNK_SIZE defined as 1MB
-pub static MAX_CHUNK_SIZE: u32 = 1024*1024;
+pub const MAX_CHUNK_SIZE: u32 = 1024 * 1024;
 /// MIN_CHUNK_SIZE defined as 1KB
-pub static MIN_CHUNK_SIZE: u32 = 1024;
+pub const MIN_CHUNK_SIZE: u32 = 1024;
+
 /// Helper function to XOR a data with a pad (pad will be rotated to fill the length)
 /// ### Usage
-///    let data = vec![1u8, 2u8, 3u8, 4u8, 5u8];  
-///    let pad = vec![8u8, 7u8];  
+///    let data = vec![1u8, 2u8, 3u8, 4u8, 5u8];
+///    let pad = vec![8u8, 7u8];
 ///    let xor_result = xor(&data, &pad);
-/// 
```

Comment:
Sorry, that specific one was during a rebase and didn't notice the code sample wasn't wrapped.

You should surround code with \`\`\` if you want it formatted with appropriate spacing. They'll also get tested that they compile and run :) (if you want to just test compile but not run, use \`\`\`{rust, no_run}